android/src/fdroid/play/listings/fr-FR/release-notes.txt

```diff
@@ -1,10 +1,11 @@
-• Nouvelles données OpenStreetMap au 27 février
-• Les articles Wikipedia sont désormais cliquables et visibles pour les signets
-• Correction de plusieurs plantages
-• Amélioration aide et dialogues des POI sélectionnés
-• Correction d'heures d'ouverture complexes non affichées
-• Recherche corrigée et améliorée
-• Ajout spécialistes de soins de santé
-• Ajout des tours de communication, des générateurs solaires, éoliens, gaziers et hydroélectriques
-• Amélioration traductions
-…Plus : omaps.app/news
+• New OSM Planet data as of March 26
+• Experimental Lane assistant
+• Fix wrong location arrow position during navigation
+• Fix bookmark's color chooser for Right-To-Left languages
+• Properly display speed and distance in navigation mode
+• Fix API to select a point on the map
+• Many new shop types, including cannabis one :)
+• Add money transfer, bicycle charging stations, power plant
+• Made park, forest, and scrub colors different
+• Improved ar, be, de, hu, tr translations
+…more: omaps.app/news
```

Comment:
* Nouvelles données Openstreetmap du 26 Mars
* Indication expérimentale des voies de bifurcation
* Correction position flèche de localisation en navigation
* Correction sélecteur de couleur des signets
* Correction affichage vitesse et distance en navigation
* Correction API pour sélectionner un point sur la carte
* Beaucoup de nouveaux types de magasins, y compris cannabis :)
* Ajout transfert d'argent, stations de recharge de vélos et centrales électriques
* Différenciation des couleurs des parcs, des forêts et des broussailles
* Amélioration traductions
…Plus : omaps.app/new